.github/workflows/docs-sched-rebuild.yaml

```diff
@@ -71,6 +71,46 @@ jobs:
             echo "Git adding ${i}"
             git add "${i}"
           done
+      - name: Check or create dot-no-jekyll file
+        run: |
+          if [ -f ".nojekyll" ]; then
+            echo "The dot-no-jekyll file already exists."
+            exit 0
+          fi
+          touch .nojekyll
+          git add .nojekyll
```

Comment:
doesn't the .nojekyll file get added by the `sphinx.ext.githubpages` https://github.com/NVIDIA-Merlin/models/blob/1abae0e2b60010c2bcd8cdb308b09cd98fdc3e75/docs/source/conf.py#L65  - or is this not playing well with sphinxmultiversion?

Reply:
Ah, look at that.  I didn't realize that would attempt to add the `.nojekyll` file.  Regardless of my ignorance, the file is added in the directory for each version, so `main/.nojekyll` and `v0.2.0/.nojekyll` and not in the root of the repo.  It was simple enough to add.